completion.go

```diff
@@ -105,12 +115,20 @@ var O1SeriesModels = map[string]struct{}{
 	O1Preview20240912: {},
 }
 
+// O3SeriesModels List of new Series of OpenAI models.
+var O3SeriesModels = map[string]struct{}{
+	O3Mini:         {},
+	O3Mini20250131: {},
+}
```

Comment:
We'll need to bookkeep this map forever and ever, could we use the `strings.HasPrefix("o3-")` instead?